apps/storefront/src/hooks/dom/useCartToQuote.ts

```diff
@@ -58,6 +59,20 @@ const useCartToQuote = ({ setOpenPage, cartQuoteEnabled }: MutationObserverProps
 
   const { pathname } = window.location;
 
+  useEffect(() => {
+    const showRejectedAccountTip = () => {
+      if (![CHECKOUT_URL].includes(pathname)) return;
+
+      if (companyStatus !== CompanyStatus.REJECTED) return;
+
+      if (checkIsInPage(CHECKOUT_URL)) {
```

Comment:
isn't this check superfluous? we've already determined in line 64 that we are indeed in the checkout page? Also, why not use this type of check in line 64?